data/json/items/fluff.json

```diff
@@ -97,6 +111,326 @@
+        "color": "blue",
+        "weight": 20
+      },
+      {
+        "id": "jigsaw_30",
+        "name": { "str": "Portland Head Light jigsaw puzzle" },
+        "description": "A jigsaw puzzle cardboard box containing 1000 cardboard pieces that when connected together properly forms the beautiful front view of the Portland Head Light in Maine.  Its picturesque setting overlooking the Atlantic Ocean is outstanding!",
+        "color": "blue",
+        "weight": 20
+      },
+      {
+        "id": "jigsaw_31",
+        "name": { "str": "Taj Mahal jigsaw puzzle" },
+        "description": "A jigsaw puzzle cardboard box containing 1000 cardboard pieces that when connected together properly forms the background view of the Taj Mahal mausoleum in Agra, India.",
+        "color": "blue",
+        "weight": 20
+      },
+      {
+        "id": "jigsaw_32",
+        "name": { "str": "Eiffel Tower jigsaw puzzle" },
+        "description": "A jigsaw puzzle cardboard box containing 1000 cardboard pieces that when connected together properly forms the beautiful front view of the Eiffel Tower in Paris, France.",
+        "color": "blue",
+        "weight": 20
+      },
+      {
+        "id": "jigsaw_33",
+        "name": { "str": "The Statue of Liberty jigsaw puzzle" },
+        "description": "A jigsaw puzzle cardboard box containing 1000 cardboard pieces that when connected together properly forms the unique, beautiful front view of the Statue of Liberty, in New York.",
+        "color": "blue",
+        "weight": 20
+      }
+    ]
+  },
+  {
+    "id": "tic_tac",
+    "type": "GENERIC",
+    "category": "other",
+    "name": { "str": "tic-tac-toe set" },
+    "description": "Not really a set, just a sheet of paper with a big '#' drawn at the center, alongside three buttons of a specific color, three others from a different color and a proper container for storing it properly.  You can play tic tac toe indefinitely with that!",
+    "weight": "100 g",
+    "volume": "500 ml",
+    "material": [ "paper" ],
+    "symbol": "?",
+    "color": "white",
+    "flags": [ "NO_REPAIR" ],
+    "use_action": [ "PLAY_GAME" ]
+  },
+  {
+    "id": "makeshift_dominoes",
+    "type": "GENERIC",
+    "category": "other",
+    "name": { "str": "makeshift dominoes set" },
+    "description": "Just a proper number of rectangular pieces of cardboard, with dots on them so you can use it to play dominoes.  It has also a proper container for storing them all in place.",
+    "weight": "100 g",
+    "volume": "500 ml",
+    "material": [ "cardboard" ],
+    "symbol": "?",
+    "color": "white",
+    "flags": [ "NO_REPAIR" ],
+    "use_action": [ "PLAY_GAME" ]
+  },
+  {
+    "id": "checkboard",
+    "type": "GENERIC",
+    "category": "other",
+    "name": { "str": "checkboard" },
```

Comment:
I believe it is some form of abbreviation, because if you search for "checkboard" in Google, there will be images of checkboards, but idk, though, I'll probably change it soon when I get some free time.